docs/cloud/cloud-org/cloud-org-upgrade-to-paid.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade your organization to paid
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 4
+---
+
+# How do I upgrade my organization to paid?
```

Comment:
paid what?

Reply:
I suspect the devs have added a payment system to Cloud?

I haven't seen this yet, so is this a method of purchasing credit for database usage? In this case this could be more descriptive:

"How do I purchase credit for my FeatureBase Cloud databases?"

Alternatively, if the user is adding a C/Card for ongoing payments, then maybe:

"How do I add a credit-card to my FeatureBase Cloud account?"

Reply:
Changed